### PR TITLE
Added AppiumSetValueOptions to hide keyboard

### PR DESCRIPTION
**Pull request by amuthansakthivel**
Added AppiumSetValueOptions to hide keyboard if needed after setting value to textbox.

It is a painful process to hide keyboard after every setValue. Hiding keyboard is not needed always. It is needed only if the screen size is small or element is at bottom of the screen. 

So we want to leave the choice to users.

$(element).setValue(withText("abcd123").hideKeyboard());
$(element).setValue(withText("abcd123"));
$(element).setValue("abcd123");

@BorisOsipov 

## Checklist
- [x] Checkstyle and unit tests are passed locally with my changes by running `gradlew check` command
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/test/java/integration/android/AndroidSetValueOptionsTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class AndroidSetValueOptionsTest extends BaseSwagLabsAndroidTest {
+
+  @Test
+  void testHideKeyboardForAndroid() {
+    SelenideAppium.launchApp();
+
+    $(xpath("//android.view.ViewGroup[@content-desc='open menu']/android.widget.ImageView"))
+      .shouldBe(visible).click();
+    $(byText("Log In")).shouldBe(visible).click();
```
Comment (BorisOsipov) [inline src/test/java/integration/android/AndroidSetValueOptionsTest.java:22]:
```suggestion
    SelenideAppium.openAndroidDeepLink("mydemoapprn://login", "com.saucelabs.mydemoapp.rn");
```

**Comment by asolntsev** [comment]
@amuthansakthivel The change is good technically, but I am thinking usability...
I mean, instead if writing
```java
$.setValue(withText("abcd").hideKeyboard());
```

Isn't it simpler to just write `setValue` and `hideKeyboard` separately?

```java
$.setValue("abcd").hideKeyboard();
```

The only thing we need to achieve that is make method `$` return something like `SelenideAppiumElement` instead of `SelenideElement`. Probably we should do it anyway. 
Then users will need to import `com.codeborne.selenide.appium.SelenideAppium.$` instead of `com.codeborne.selenide.Selenide.$`. Seems easy.

**Comment by amuthansakthivel** [comment]
> @amuthansakthivel The change is good technically, but I am thinking usability... I mean, instead if writing
> 
> ```java
> $.setValue(withText("abcd").hideKeyboard());
> ```
> 
> Isn't it simpler to just write `setValue` and `hideKeyboard` separately?
> 
> ```java
> $.setValue("abcd").hideKeyboard();
> ```
> 
> The only thing we need to achieve that is make method `$` return something like `SelenideAppiumElement` instead of `SelenideElement`. Probably we should do it anyway. Then users will need to import `com.codeborne.selenide.appium.SelenideAppium.$` instead of `com.codeborne.selenide.Selenide.$`. Seems easy.

But @asolntsev, when are we planning to implement that class. This SelenideAppiumElement class should also https://github.com/selenide/selenide-appium/issues/106
https://github.com/selenide/selenide-appium/issues/105
https://github.com/selenide/selenide-appium/issues/86

**Comment by asolntsev** [comment]
@amuthansakthivel Please take a look: I added method `$.hideKeyboard()` in PR https://github.com/selenide/selenide-appium/pull/131

**Comment by amuthansakthivel** [comment]
@asolntsev - i am closing this PR then

**Pull request closed without merging**